protocols/plaintext/src/lib.rs

```diff
@@ -55,3 +70,130 @@ impl<C> OutboundUpgrade<C> for PlainTextConfig {
     }
 }
 
+#[derive(Clone)]
+pub struct PlainText2Config {
+    pub peer_id: PeerId,
```

Comment:
I suppose that's the remote's `PeerId`? If so that should be removed.

Reply:
From the other conversation, this seems to be our local `PeerId`. I think it should be automatically generated from the `pubkey` then, and not provided by the configuration.

Reply:
Deleted the `PeerId` via https://github.com/libp2p/rust-libp2p/pull/1236/commits/abe8499e659374d878f7133e3473f98eb124f7cd